Description: Don't install the udev file into /usr/share
 We install the udev rules into /etc/udev/ so there's no need for a
 duplicate copy in /usr/share/
Author: Jonathan McDowell <noodles@earth.li>
Last-Update: 2017-09-25
diff --git a/Makefile.am b/Makefile.am
index 930a3073..249438a7 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -13,8 +13,7 @@ nobase_dist_pkgdata_DATA = \
 	contrib/libdcc/dcc_stdio.c \
 	contrib/libdcc/dcc_stdio.h \
 	contrib/libdcc/example.c \
-	contrib/libdcc/README \
-	contrib/60-openocd.rules
+	contrib/libdcc/README
 
 SUBDIRS =
 DIST_SUBDIRS =
